tests/xor_lookups/models.py

```diff
@@ -0,0 +1,23 @@
+"""
+XOR lookups
+
+To perform an XOR lookup, or a lookup that combines ANDs and XORs, combine
+``QuerySet`` objects using ``&`` and ``^`` operators.
+
+Alternatively, use positional arguments, and pass one or more expressions of
+clauses using the variable ``django.db.models.Q``.
+"""
+
+from django.db import models
+
+
+class Number(models.Model):
+    num = models.IntegerField()
+    other_num = models.IntegerField(null=True)
+    another_num = models.IntegerField(null=True)
+
+    def __str__(self):
+        return str(self.num)
+
+    class Meta:
+        ordering = ('num', )
```

Comment:
Do we need a `Meta.ordering`?

Reply:
Probably not, I used models from another test folder.

I'm not ordering manually in the tests however, and I'm not sure what might happen on another backend that I can't test immediately. It doesn't change much, so I'm just going to leave it?